test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -761,3 +761,110 @@ Feature: Bind values from a secret referred in backing service resource
             """
             bla
             """
+
+    Scenario: Inject binding to an application from a Secret resource referred as service
+        Given The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: provisioned-secret-1
+            stringData:
+                username: foo
+                password: bar
+            """
+        * Generic test application "myaop-provision-srv" is running
+        When Service Binding is applied
+          """
+          apiVersion: binding.operators.coreos.com/v1alpha1
+          kind: ServiceBinding
+          metadata:
+              name: bind-provisioned-service-1
+          spec:
+              services:
+              - group: ""
+                version: v1
+                kind: Secret
+                name: provisioned-secret-1
+              application:
+                name: myaop-provision-srv
+                group: apps
+                version: v1
+                resource: deployments
+          """
+        Then Service Binding "bind-provisioned-service-1" is ready
+        And jq ".status.secret" of Service Binding "bind-provisioned-service-1" should be changed to "provisioned-secret-1"
+        And Content of file "/bindings/bind-provisioned-service-1/username" in application pod is
+            """
+            foo
+            """
+        And Content of file "/bindings/bind-provisioned-service-1/password" in application pod is
+            """
+            bar
+            """
+
+    Scenario: Inject binding to an application from a Secret resource created later referred as service 
+        Given Generic test application "myaop-provision-srv" is running
+        When Service Binding is applied
+          """
+          apiVersion: binding.operators.coreos.com/v1alpha1
+          kind: ServiceBinding
+          metadata:
+              name: bind-provisioned-service-1
+          spec:
+              services:
+              - group: ""
+                version: v1
+                kind: Secret
+                name: provisioned-secret-1
+                id: one
+              - group: ""
```

Comment:
do we really need two services for this scenario? If not let's keep the setup minimal.

Reply:
There was a comment from Igor about what happens if there are multiple Secret resources.
I tried to combine it. Looks like it's confusing, I will create a separate test case for that.